app/src/main/java/com/deuvox/deuvoxapp/feature/main/splashscreen/SplashScreenActivity.kt

```diff
@@ -0,0 +1,41 @@
+package com.deuvox.deuvoxapp.feature.main.splashscreen
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.lifecycle.ViewModelProvider
+import com.deuvox.deuvoxapp.base.activity.BaseActivity
+import com.deuvox.deuvoxapp.databinding.SplashScreenBinding
+import com.deuvox.deuvoxapp.feature.main.ExampleMainActivity
+import com.deuvox.deuvoxapp.feature.main.di.DaggerSplashScreenComponent
+import kotlinx.coroutines.MainScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+class SplashScreenActivity : BaseActivity<SplashScreenBinding>() {
+    private val viewModel by lazy {
+        ViewModelProvider(this, viewModelFactory)[SplashScreenViewModel::class.java]
+    }
```

Comment:
please use viewmodel delegate, pull it from `sprint1-develop`
use it like this
```suggestion
    private val viewModel by viewModel<SplashScreenViewModel>()
```

Reply:
Built-in delegate or custom delegate ?

Reply:
custom delegate, from here > #5